apps/modernization-ui/src/design-system/table/preferences/useColumnPreferences.tsx

```diff
@@ -43,6 +43,7 @@ const reducer = (current: State, action: Action): State => {
             return { initial: action.preferences, preferences: action.preferences };
         }
         case 'save': {
+            console.log('saving', current);
```

Comment:
console.log here we might want to remove

Reply:
fixed